backend/src/main/java/io/debezium/configserver/rest/client/JolokiaClient.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.configserver.rest.client;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import javax.management.MalformedObjectNameException;
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.inject.Inject;
+
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+import org.jolokia.client.BasicAuthenticator;
+import org.jolokia.client.J4pClient;
+import org.jolokia.client.exception.J4pException;
+import org.jolokia.client.request.J4pReadRequest;
+import org.jolokia.client.request.J4pReadResponse;
+
+@ApplicationScoped
+public class JolokiaClient {
+    public static final int DEFAULT_JOLOKIA_PORT = 8778;
+
+    public static final String PROPERTY_JOLOKIA_CLIENT_URI = "jolokia.client.uri";
+
+    @ConfigProperty(name = PROPERTY_JOLOKIA_CLIENT_URI)
+    private String configJolokiaUrl;
+    @Inject
+    JolokiaAttributes jolokiaAttributes;
```

Comment:
This is also related to my previous comment about injection. Constructor or setter injection should make it easier to mock and/or test this code. @Naros @jpechane wdyt?

Reply:
See my earlier comment about `QuarkusMock` and how this is really irrelevant as their style works the same regardless of the method used, making building tests a breeze for this kind of thing.